docs-requirements.txt

```diff
@@ -0,0 +1,4 @@
+pydata-sphinx-theme>=0.3.1
+Sphinx>=2.0.1
+nbconvert>=5.5.0
+nbsphinx>=0.4.2
```

Comment:
Separated this out from `dev-requirements.txt` so that `.readthedocs.yml` can install these deps without having to install the others.

I didn't confirm that this change is necessary in order to fix the bug, it just felt like a good way to reduce complexity and speed up the build, to only install the minimum required deps in each RTD build